packages/slackctl/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack-wrench/slackctl",
-  "version": "1.0.0-alpha.11",
+  "version": "1.1.0",
```

Comment:
this package doesn't really work today, so not sure about bumping from alpha. care amount low though, we can always build-out over time